### ISSUE TITLE
Account authentication

### ISSUE THREAD

**Issue by rossyman** [svelteclub/club#9]
Developers should be able to log in to their account based on credentials supplied by the registration logic handled in #8.

Blocked by #8

### Acceptance Criteria
- Authentication logic should be based on the same architectural decisions from #8, preferably in a way that doesn't impede flexibility, such as hexagonal architecture
- Authentication logic should mimic the live beta site
- After authenticating, the user should redirected to the homepage and be shown some persistent information; potentially their handle in the navigation, linking to an account details page (Which will be handled in another task)
